docs/outputs.md

```diff
@@ -48,3 +48,105 @@ which will produce a ```png``` image per dump suitable for encoding into a movie
 ## Visualization software
 
 Both [ParaView](https://www.paraview.org/) and [VisIt](https://wci.llnl.gov/simulation/computer-codes/visit/) are capable of opening and visualizing Parthenon graphics dumps.  In both cases, the ```.xdmf``` files should be opened.  In ParaView, select the "XDMF Reader" when prompted.
+
+## Preparing outputs for ```yt```
+
+Parthenon HDF5 outputs can be read with the python visualization library
+[yt](https://yt-project.org/) as certain variables are named during when adding
+fields via ```StateDescriptor::AddField```. Variable names are added as a
+```std::vector<std::string>``` in the variable metadata. These labels are only
+used for output to HDF5. 4D variables are named with a list of names for each
```

Comment:
Are 5D variables supported? Are they simply rasterized?

Reply:
Currently not supported. The frontend is built around identifying either the conserved or primitive variables in  3D space, either being packed together in one 4D array, split up into individual 3D arrays, or some combination.

I'm not sure how we should interpret 5D data. Should the extra dimension be different species (`yt` as a whole doesn't really deal with multifluid) or should we interpolate them?

Reply:
I was more thinking about, e.g., tensor quantities. i.e., the Reynolds stress for example. But multiple species (which I think we might implement as sparse variables) would be another example.

Reply:
Unfortunately, `yt` doesn't have any way to navigate tensors seemlessly. Each component would have to be named. From the way the output by Parthenon is implement and how I've implemented the `yt` frontend works currently, I think that would work without little to no modification to the `yt` frontend. 

We can keep that in mind if we want a separate `yt` test in `Parthenon` once the `yt` frontend is firmly solidified.